backend/src/app/modules/quiz/quiz.service.ts

```
@@ -50,9 +50,48 @@
+                    moduleItemId: quiz?.moduleItemId ?? ""
+                }
+            },
+            include: {
+                QuizProgress: true
+            }
+        })
+
+        result.progress = quizProgress?.QuizProgress ?? null
+    }
+
+    delete result.moduleItem.module
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)